popSimManu.tex

```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
-do reveal some broad generalizations about relative performance. For
-example, at the task of estimating $N(t)$ trajectories for single human
+benchmarking comparisons have been limited, but nevertheless
+reveal some informative comparisons. For
+example, at the task of estimating $N(t)$ trajectories for simulations of single human
 populations, we find that the sequence-based methods (\MSMC and \smcpp)
 perform somewhat better overall---at least for mid-range values of
 $t$---than the site frequency spectrum-based method (\stairwayplot)
-(Figure~\ref{fig:n_t_ragsdale} and Figure~\ref{fig:n_t_gutenkunst}), which
-tends to over-estimate oscillations in size.  By contrast, in the case of
-\emph{D.~melanogaster}, where linkage disequilibrium is considerably
-reduced, \stairwayplot outperforms the sequence-based methods
-(Figure~\ref{fig:n_t_sheehan}). In the two-population setting for human
+(Figures~\ref{fig:n_t_ragsdale} and~\ref{fig:n_t_gutenkunst}), which
+tends to over-estimate the size of oscillations.  By contrast,
+\stairwayplot outperforms the sequence-based methods
+on simulations of \textit{D.~melanogaster} populations,
+in which linkage disequilibrium is considerably reduced (Figure~\ref{fig:n_t_sheehan}).
+In simulations of two human populations
 (Figure~\ref{fig:IM_popn_human}), most methods do reasonably well at
 reconstructing the YRI history, but struggle with the more complex CEU
 history, in large part because of the restriction of constant sizes per
 population.  An exception is \smcpp, which does not have the same
 restrictions on its inferred history, and as a result does somewhat better
 with the CEU history but tends to overfit the YRI history.  The results for
-the two-population {\em Drosophila} case (Figure~\ref{fig:two_popn_fly})
+the two-population \textit{D.~melanogaster} case (Figure~\ref{fig:two_popn_fly})
 are generally similar.  
 
 Altogether, these preliminary experiments highlight
 the utility of \stdpopsim for comparing a variety of inference methods on
 the same footing, under a variety of different demographic models.
 In addition, the ability of \stdpopsim to generate data with and without significant features, such
-as the genetic map or population size change (e.g., Figure \ref{fig:n_t_HomSap_constant}), allows
+as a genetic map or population size change (e.g., Figure \ref{fig:n_t_HomSap_constant}), allows
 investigation of the failure modes of popular methods.
 Moreover the comparison of methods across the various genome organizations, genetic maps,
 and demographic histories of different organisms, provides valuable information
 about how methods might perform on non-human systems.
 
 Finally, comparison of results across methods or simulation runs
 provides an estimate of inference uncertainty, analogous to parametric
-bootstrapping.
+bootstrapping,
+especially since different methods are likely vulnerable to model misspecification
+in different ways.
```

Comment:
clarifying what we mean here by 'bootstrapping' (across runs is like bootstrapping, but so is using different methods, although less clearly)